main.js

```diff
@@ -0,0 +1,24 @@
+var menu = document.querySelector(".menu-mobile");
```

Comment:
The whole script (all the contents of .js file) should be wrapped in a so-called IIFE (immediately-invoked function expression) so as not to pollute global scope with custom variables. There's a nice explanation on Flavio Copes blog: https://flaviocopes.com/javascript-iife/ (I generally recommend his articles a lot, they are short, concise and very "meaty" :) )

Additionally, it's considered a good practice to add a listener to `DOMContentLoaded`: https://flaviocopes.com/dom-ready/
https://www.youtube.com/watch?v=m1DH8ljCqzo&ab_channel=dcode

So, the whole thing would look like this:

```(function () {
    document.addEventListener("DOMContentLoaded", function () {
         // (your scripts here)
    });
})();
```

Reply:
I don't understand this article :( The IIFE conserns the whole script or a single function?

Reply:
The whole script. :) Same with DOMContentLoaded. Iife is used so that your variables do not get defined outside of the scope of your script. Without it, if you, say, define `const myNumber = 1;`, it'd be available on the global object. In the case of browser, `window`. You could then call `window.myNumber` and it'd be equivalent to calling `myNumber`. We do not want to affect global object in any way, it could lead to some problems. That's why we create a so-called `namespace`, wrapping the whole script in an iife, a function that is created and immediately invoked by itself. It protects your script like a bubble. :)

Your code will probably work perfectly without these wrappers but it's good practice to use them. :)

Reply:
It's clear now :) You are a better teacher than Flavio C.